---
permalink: /
title: "Research"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

### I am interested in questions about the evolution of plants, genes, plant genomes and transcriptomes

**Why are the structures of spiral flowers flexible?**
<a href="https://www.nature.com/articles/nplants2015188/figures/3"><img src="../images/41477_2016_Article_BFnplants2015188_Fig3.png" align="right" width="100"></a> 
Using *Nigella damascena* (love-in-a-mist) as a model system, we revealed that the expression profiles of floral organ identify genes are dymatic, and the genes regulate the boundaries between different types of floral organs by forming a complex regulatory network. Notably, the *AGL6*-lineage member *NdAGL6* is an A-function gene determining the identity of sepal and petal, rather than the *AP1*-lineage genes as in *Arabidopsis thaliana*. These result have been published in *Nature Plants*. [pubmed](https://pubmed.ncbi.nlm.nih.gov/27250746/)

**How do gene family sizes vary among related plant species?**
<a href="https://pubmed.ncbi.nlm.nih.gov/30239695/#&gid=article-figures&pid=scfigsc-5-uid-4"><img src="../images/GBE_fig5.png" align="right" width="100"></a> 
Using the Solanaceae family as a model and Pfam domain families as a proxy for gene families, our findings reveal complex relationships between variation in gene family size, gene functions, duplication mechanism, and evolutionary rate: 1) genes in highly variable families have high turnover rates and tend to be involved in processes that have diverged between Solanaceae species, whereas genes in low-variability families tend to have housekeeping roles; 2) genes in high and low-variability gene families tend to be duplicated by tandem and whole genome duplication, respectively; 3) genes duplicated by different mechanisms experience different selection pressures. In addition, we found that a substantial number of pseudogenes are actually products of pseudogene duplication, contrary to the expectation that most plant pseudogenes are remnants of once-functional duplicates. These results can be available on *Genome Biol Evol*. [pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7852129/)

**What factors influence read coverage in a genome assembled using short reads?**


**How well can the gene expression data be used to predict metabolic pathway memberships?**





