---
permalink: /
title: "Research"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

## I am generally interested in questions about the evolution of plants, genes, plant genomes and transcriptomes, and the predictive biology using machine learning. 


**How do transcriptome and methylome contribute to trait prediction?**
<a href="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-024-50701-6/MediaObjects/41467_2024_50701_Fig6_HTML.png?as=webp"><img src="../images/Fig. 1 Flow_chart_v7.jpg" align="right" width="350"></a> 
Transcriptome- and methylome-based models had performances comparable to those of genome-based models in *Arabidopsis thaliana*, but identified different benchmark genes. Gene contributions to flowering time prediction are accession-dependent and distinct genes contribute to trait prediction in different genotypes. Models integrating multi-omics data perform best and reveal known and additional gene interactions, extending knowledge about existing regulatory networks underlying flowering time determination. These results have been published in ***Nat Commun***. [pubmed](https://pubmed.ncbi.nlm.nih.gov/39127735/)

**Why are the structures of spiral flowers flexible?**
<a href="https://www.nature.com/articles/nplants2015188/figures/3"><img src="../images/41477_2016_Article_BFnplants2015188_Fig3.png" align="right" width="150"></a> 
Using *Nigella damascena* (love-in-a-mist) as a model system, we revealed that the expression profiles of floral organ identify genes are highly dynamic, and the genes regulate the boundaries between different types of floral organs by forming a complex regulatory network. Notably, the *AGL6*-lineage member *NdAGL6* is an A-function gene determining the identity of sepal and petal, rather than the *AP1*-lineage genes as in *Arabidopsis thaliana*. These results have been published in ***Nat Plants***. [pubmed](https://pubmed.ncbi.nlm.nih.gov/27250746/)


**How can we measure Arabidopsis fitness efficiently?**
<a href="https://nph.onlinelibrary.wiley.com/cms/asset/2c994acd-68fa-42a2-a374-e3842ad4ad86/nph18056-fig-0002-m.png"><img src="../images/nph18056-fig-0002-m.jpg" align="right" width="150"></a> Using the Faster Region-based Convolutional Neural Network (R-CNN) algorithm, we established the image recognition models for Arabidopsis seeds and fruits, which yielded near perfect counts (r2 = 0.9996 and 0.980, respectively). These results have been published in ***New Phytol***.[pubmed](https://pubmed.ncbi.nlm.nih.gov/35218008/)

**How well can the gene expression data be used to predict metabolic pathway memberships?**
<a href="https://nph.onlinelibrary.wiley.com/cms/asset/0d5858f8-88fa-4fa8-8eb5-fd14301eb72c/nph17355-fig-0006-m.jpg"><img src="../images/nph17355-fig-0006-m.jpg" align="right" width="350"></a> 
To optimize the use of gene expression data to predict plant metabolic pathway memberships, we explored >600 tomato expression data combinations, three strategies for predicting memberships in 85 pathways. We found that optimal predictions for different pathways require distinct data combinations indicative of pathway functions. Unsupervised learning performed better than supervised approaches, while gene-to-pathway expression similarities led to prediction models that outperformed those based simply on expression levels. Predictions for experimental validated genes is significantly better than that for genes without experimental evidence, demonstrating the importance of data quality. Our study highlights the need to extensively explore expression-based features and prediction strategies to maximize the accuracy of metabolic pathway membership assignment. These results have been published in ***New Phytol***. [pubmed](https://pubmed.ncbi.nlm.nih.gov/33749860/)

**How do gene family sizes vary among related plant species?**
<a href="https://pubmed.ncbi.nlm.nih.gov/30239695/#&gid=article-figures&pid=scfigsc-5-uid-4"><img src="../images/GBE_fig5.png" align="right" width="150"></a> 
Using the Solanaceae family as a model and Pfam domain families as a proxy for gene families, our findings reveal complex relationships between variation in gene family size, gene functions, duplication mechanism, and evolutionary rate: 1) genes in highly variable families have high turnover rates and tend to be involved in processes that have diverged between Solanaceae species, whereas genes in low-variability families tend to have housekeeping roles; 2) genes in high and low-variability gene families tend to be duplicated by tandem and whole genome duplication, respectively; 3) genes duplicated by different mechanisms experience different selection pressures. In addition, we found that a substantial number of pseudogenes are actually products of pseudogene duplication, contrary to the expectation that most plant pseudogenes are remnants of once-functional duplicates. These results can be available in ***Genome Biol Evol***. [pubmed](https://pubmed.ncbi.nlm.nih.gov/30239695/)

**What are the factors influencing read coverage and misassembly in a genome assembled using short reads?**
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7852129/bin/12864_2021_7397_MOESM4_ESM.pdf"><img src="../images/BMC_figS4.png" align="right" width="150"></a> 
In tomato, 0.6% (5.1 Mb) and 9.7% (79.6 Mb) of short-read based assembly had significantly higher and lower coverage compared to background, respectively. We first established machine learning models capable of predicting genomic regions with variable coverages and found that high coverage regions tend to have higher simple sequence repeat and tandem gene densities compared to background regions. We also found that 27.8% (1.41 Mb) of high coverage regions were potentially misassembled of duplicate sequences, compared to 1.4% in background regions. In addition, misassembled, high coverage regions tend to be flanked by simple sequence repeats, pseudogenes, and transposon elements. Our study provides insights on the causes of variable coverage regions and a quantitative assessment of factors contributing to plant genome misassembly when using short reads. These results have been published in ***BMC Genomics***. [pubmed](https://pubmed.ncbi.nlm.nih.gov/33530937/)





